plugins/remark-lint-validate-internal-links/index.js

```diff
@@ -0,0 +1,204 @@
+import { accessSync, constants, readFileSync } from 'fs';
+import { join } from 'path';
+import { lintRule } from 'unified-lint-rule';
+import { visit } from 'unist-util-visit';
+import { generated } from 'unist-util-generated';
```

Comment:
*issue*: 'generated' is not used anywhere, need to be removed.

Reply:
fixed